docker/ml-api-adapter/default_iso.json

```diff
@@ -0,0 +1,201 @@
+{
+  "API_TYPE": "iso20022",
+  "ORIGINAL_PAYLOAD_STORAGE": "",
+  "PAYLOAD_CACHE": {
+    "enabled": true,
+    "type": "redis-cluster",
+    "connectionConfig": {
```

Comment:
Do we need this file? I think we can just overrided using env variables in docker compose if we need any iso specific config

Reply:
`default_iso.json` was added for spinning up iso container for integration tests only.